src/components/action-bar/action-bar.scss

```diff
@@ -50,6 +50,10 @@
   }
 }
 
+:host([layout="horizontal"][expand-disabled]) ::slotted(calcite-action-group:last-of-type) {
+  @apply border-r-0;
```

Comment:
hmm instead of border-right should we be using border-block-end or border-inline-end? Does this look ok in RTL?

Reply:
Ha, yeah it looks good in `rtl` because we were only applying the border with `border-r`. I'll update the file to use logical css properties.

Reply:
Updated @driskull